src/completers/git.zsh

```diff
@@ -372,15 +367,14 @@ _fzf_complete_git-commit-messages_post() {
         '$_fzf_complete_awk_functions'
```

Comment:
This part no longer uses `$fzf_complete_awk_functions` which should be removed as well.

Reply:
Apart from that, this procedure could be rewritten to something like:
```zsh
local message=$(awk '{
    sub($0, /[^ ]*  /, "")
    print
}')
```